### PR TITLE
Add Verdant Dread (YDSK)

### PR DESCRIPTION
**Pull request by TwentyToedToad**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/verdant_dread.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Verdant Dread
+ManaCost:1 G
+Types:Enchantment
+T:Mode$ ChangesZone | ValidCard$ Card.Self,Card.namedVerdant Dread | Origin$ Any | Destination$ Battlefield | Execute$ TrigManifestDread | TriggerDescription$ Whenever CARDNAME or another card named CARDNAME enters, manifest dread.
+SVar:TrigManifestDread:DB$ ManifestDread
+A:AB$ MakeCard | Cost$ 3 G G | Conjure$ True | Name$ Verdant Dread | Zone$ Battlefield | SorcerySpeed$ True | SpellDescription$ Conjure a card named CARDNAME onto the battlefield. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/verdant_dread.txt:6]: the "named X" parts are constants

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/verdant_dread.txt]: the first instance was correct :p

Comment (TwentyToedToad) [reply forge-gui/res/cardsfolder/upcoming/verdant_dread.txt:6]: I modified it back to original, is there something else that needs changed?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/verdant_dread.txt:6]: uhh I think you misunderstood again
I'll edit it correctly, let me know if you need more details

**Review by tool4ever — APPROVED** [review]

**Pull request merged**